search/api.py

```diff
@@ -27,6 +27,17 @@ class NoSearchEngineError(Exception):
     pass
 
 
+class QueryParseError(Exception):
+    """QueryParseError will be thrown if the query is malformed.
+
+    With the removal of quotes from RESERVED_CHARACTERS, it's possible
+    to get a parsing error back from Elastic. This error wraps that so
```

Comment:
This is a good explanatory comment for this PR, but is not required in the code, since the maintainers that come after us won't be concerned with what `RESERVED_CHARACTERS` previously contained.

Could you replace this comment with some examples of malformed queries instead?

Reply:
Nudge @ciuin :)